lib/UtapiClient.js

```diff
@@ -126,7 +126,7 @@ class UtapiClient {
         this.enabledOperationCounters = [];
         this.disableClient = true;
 
-        if (config) {
+        if (config && !config.disableClient) {
```

Comment:
should it be this.diableClient?

Reply:
`config.disableClient` is correct. The intent is to allow config to be passed while not enabling it as cloudserver always instantiates a client at startup and relies on the config being undefined to disable it. Context is that I need to pass the tls certs into the v2 client and would like to keep the logic on cloudservers side simple

Reply:
Cool